tests/deepsparse/loggers/metric_functions/template_functions/template_logs/image_classification_logs.txt

```diff
@@ -0,0 +1,6 @@
+identifier:image_classification/pipeline_inputs.images__image_shape, value:{'channels': 3, 'dim_0': 224, 'dim_1': 224}, category:MetricCategories.DATA
+identifier:image_classification/pipeline_inputs.images__mean_pixels_per_channel, value:{'channel_0': 1.0, 'channel_1': 1.0, 'channel_2': 1.0}, category:MetricCategories.DATA
+identifier:image_classification/pipeline_inputs.images__std_pixels_per_channel, value:{'channel_0': 0.0, 'channel_1': 0.0, 'channel_2': 0.0}, category:MetricCategories.DATA
+identifier:image_classification/pipeline_inputs.images__fraction_zeros, value:0.0, category:MetricCategories.DATA
+identifier:image_classification/pipeline_outputs.labels__predicted_classes, value:[[530], [530]], category:MetricCategories.DATA
+identifier:image_classification/pipeline_outputs.scores__predicted_top_score, value:[0.9995013475418091, 0.9995013475418091], category:MetricCategories.DATA
```

Comment:
Also, confidence here is very high. But this can happen due to inferencing on the completely unseen data distribution (just black img)